python/core/unified_api_server.py

```
@@ -1148,6 +1162,161 @@ def _build_recommendations(
     }
 
 
+def _generate_famous_player_comparisons(
+    personality_scores: Dict[str, float],
+    player_style: Dict[str, Any]
```

Ruff (0.13.3)

Unused function argument: `player_style`

(ARG001)